cellmlmanip/model.py

```diff
@@ -387,6 +387,20 @@ def get_derived_quantities(self):
                               and node.get('variable_type', '') not in ('state', 'free', 'parameter')]
         return sorted(derived_quantities, key=lambda var: var.order_added)
 
+    def get_display_name(self, var, META='https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#'):
```

Comment:
Should `META` be called `ontology` ?

Reply:
I'd prefer it if the method didn't select a particular ontology at all if you didn't specify this.

So 
- `get_display_name(self, var, ontology=None)`
- To get only terms from oxmeta, call with `get_display_name(self, var, 'https://....')`
- To get any old annotation, call with `get_display_name(self, var)`

Reply:
@MauriceHendrix see previous comment above

Reply:
hhmm ok fine although my calls are going to be longer now